saleor/channel/models.py

```diff
@@ -49,6 +49,12 @@ class Channel(ModelWithMetadata):
     use_legacy_error_flow_for_checkout = models.BooleanField(default=True)
     automatically_complete_fully_paid_checkouts = models.BooleanField(default=False)
 
+    # automatic_completion_delay applies only when
+    # automatically_complete_fully_paid_checkouts is set to True
+    automatic_completion_delay = models.IntegerField(
```

Comment:
Can you confirm that the scale here is correct? You're using minutes to delay, which means, that I can't delay it by 15 seconds, or 45 seconds. I can pick up 0 or minute.